dev.lynith.Speedtest.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id" : "dev.lynith.Speedtest",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "speedtest",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=~/.local/share/"
```

Comment:
why do you need access to this?

Reply:
The program installs a *proprietary* binary from [Ookla's servers](https://www.speedtest.net/apps/cli) and stores it in `~/.local/share/speedtest/`

Reply:
then it's not `filesystem` you need. It's `persist`

Reply:
also does it handle architecture properly when downloading?

Reply:
> also does it handle architecture properly when downloading?

Yes